schema/data/prod/form_json/fuel.json

```diff
@@ -26,7 +26,8 @@
             "enum": ["wci 1.0", "wci 2.0", "wci 3.0"],
             "enumNames": ["WCI 1.0", "WCI 2.0", "WCI 3.0"]
           }
-        }
+        },
+        "required": ["Fuel Type", "Quantity", "Units", "Methodology"]
```

Comment:
Shouldn't these be the property names instead of their labels?

Reply:
yes